<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanControlsPanel</class>
 <widget class="QWidget" name="ScanControlsPanel">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="scanControlsGroup">
     <property name="title">
      <string>Scan Controls</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="searchTermLabel">
        <property name="text">
         <string>Search term</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="searchTermLineEdit"/>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="ignoreCaseCheckBox">
        <property name="text">
         <string>Ignore case</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="startScanButton">
        <property name="text">
         <string>Scan</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="shiftLabel">
        <property name="text">
         <string>Shift</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QSpinBox" name="shiftValueSpin">
        <property name="minimum">
         <number>-7</number>
        </property>
        <property name="maximum">
         <number>7</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QComboBox" name="shiftUnitCombo">
        <item>
         <property name="text">
          <string>Bytes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bits</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="openFileButton">
        <property name="text">
         <string>Open file/device</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="openDirButton">
        <property name="text">
         <string>Open directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="blockSizeLabel">
        <property name="text">
         <string>Block size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="blockSizeSpin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1048576</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="blockSizeUnitCombo">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>KiB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MiB</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="workerCountLabel">
        <property name="text">
         <string>Workers</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QComboBox" name="workerCountCombo"/>
      </item>
      <item row="1" column="7" colspan="2">
       <widget class="QCheckBox" name="prefillOnMergeCheckBox">
        <property name="text">
         <string>PrefillOnMerge</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="filesCountLabel">
        <property name="text">
         <string>Files</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="filesCountValueLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="searchSpaceLabel">
        <property name="text">
         <string>Search space</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="searchSpaceValueLabel">
        <property name="text">
         <string>0 B</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="scannedLabel">
        <property name="text">
         <string>Scanned</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="scannedValueLabel">
        <property name="text">
         <string>0 B</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6" colspan="2">
       <widget class="QProgressBar" name="scanProgressBar">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QLabel" name="selectedSourceLabel">
        <property name="text">
         <string>Selected</string>
        </property>
       </widget>
      </item>
      <item row="2" column="9">
       <widget class="QLabel" name="selectedSourceValueLabel">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
